### ISSUE TITLE
WakaTime Plugin Not Prompting for API Key in PyCharm Community Edition on Windows 11

### ISSUE THREAD

**Issue by Tgentil** [wakatime/jetbrains-wakatime#264]
### Description:

After following the installation steps for the WakaTime plugin in PyCharm Community Edition, the pop-up to enter the API key does not appear. I have tried disabling and enabling the plugin, as well as uninstalling and reinstalling it, but the issue persists.

### Steps to Reproduce:

1. Install PyCharm Community Edition on a Windows 11 machine.
2. Navigate to Preferences -> Plugins -> Browse Repositories...
3. Search for WakaTime and install the plugin.
4. Re-launch PyCharm.
5. Observe that there is no prompt to enter the WakaTime API key.

### Expected Behavior:

After re-launching PyCharm, a place to input my API key should appear.

### Actual Behavior:

No screens appears for entering the WakaTime API key.

### Additional Information:

- Operating System: Windows 11
- PyCharm Version: 2023.2.1
- WakaTime Plugin Version: 14.2.1

**Comment by alanhamlett** [comment]
Do you already have an api key in your common shared `~/.wakatime.cfg` file? Then it won't re-prompt.

**Comment by Tgentil** [comment]
Hi @alanhamlett,

Thank you for your quick response. You're correct; I did have an API key in my common shared `~/.wakatime.cfg` file. After I opened the issue, I uninstalled PyCharm Community and installed PyCharm Professional. I experienced the same behavior, but upon starting a new project, I noticed that my API key was already configured automatically.

Thank you for clarifying this for me, and I apologize for any inconvenience caused.

Best regards. 
(PS.: this project is amazing)
